packages/valory/skills/decision_maker_abci/behaviours/bet_placement.py

```diff
@@ -341,7 +363,7 @@ def async_act(self) -> Generator:
             tx_submitter = betting_tx_hex = None
 
             can_exchange = (
-                self.collateral_token == WXDAI
+                self.is_wxdai
```

Comment:
The addresses provided by the subgraph are not checksummed. As a result, `can_exchange` was always `False`.